CoreTimelineView/src/au/gov/asd/tac/constellation/views/timeline/clustering/ClusteringManager.java

```diff
@@ -402,7 +402,7 @@ protected void execute(final PluginGraphs graph, final PluginInteraction interac
         }
     }
 
-    @PluginInfo(tags = {"MODIFY"})
+    @PluginInfo(tags = {"LOW LEVEL"})
     public final class UpdateDimOrHidePlugin extends SimplePlugin {
```

Comment:
If you look at the plugin details closely, you'll see that it's actually updating the dimmed and visibility attributes of the graph elements and so I think 'MODIFY' is the correct tag here

Reply:
True but whats happening here is that when the timeline view is open, it is flooding the auto notes with this plugin call whenever a selection is made which isn't useful. I can put a comment explaining why this one is a low level if you like?

Reply:
@antares1470 I've changed this to include both tags and I have also added an explanation as to why the LOW LEVEL tag was added. This change will still prevent the notes view from showing the plugin call but we can see it in the Plugin Reporter if we need to.